src/Main/PlaneSliders.jsx

```diff
@@ -18,6 +18,19 @@ import '../style.css'
 function PlaneSliders(props) {
   const { service } = props
   const stateContextMain = useSelector(service, (state) => state.context.main)
```

Comment:
I threw a console.log in PlaneSliders, clicked some buttons in Main interface, and DID NOT SEE console.log.  YAY!

But, when I click to show the 2D view (z slice), it still shows 3 sliders.  When I click a slider button, 2 other sliders disappear (like originally expected) because the componenet re-rendered.  

Lets try a useSelector for `viewMode` to hide sliders when view changing buttons are clicked.  

Lesson I learned: The objects in the `context` don't necessary trigger useSelector to re-render if their children change.  It is strange that the `useSelector(service, (state) => state.context.main)` hook did update the variable when dragging the slider tick triggered a re-render.  Another mystery to ponder.

Reply:
Thanks for sharing the lesson learned. In a similar vein, clicking on the XPlane Visibility button does display the plane, but does not trigger a re-render. Then, if we call `useSelector` for `state.context.main.slicingPlanes[`x`].visible`,  we manually trigger a re-render.  The same is true for the pause button -- it does pause the plane in the current position, but does not change the button to the "play icon". 

It is not clear to me when we explicitly need to call for useSelector and why the re-render does not pick up on some updated variables.